### PR TITLE
fix link to kirschner paper

### PR DESCRIPTION
**Pull request by robitalec**
Just a small edit, the link is dead for the Kirschner paper.

Lots of link options [here](https://scholar.google.com/scholar?oi=bibs&hl=en&cluster=11065780712206344145), not sure what is preferred.

### CODE REVIEW

File: _episodes/05-memory.md [hunk 1]
```diff
@@ -423,7 +423,7 @@ can be gradually removed.
 Anything you can do to a) recognize and b) support learners in working with the
 limitations of short-term memory will improve the effectiveness of your teaching.
 
-[kirschner-paper]: http://www.cogtech.usc.edu/publications/kirschner_Sweller_Clark.pdf
+[kirschner-paper]: https://www.tandfonline.com/doi/pdf/10.1207/s15326985ep4102_1?needAccess=true
```
Comment (brownsarahm) [inline _episodes/05-memory.md:426]:
```suggestion
[kirschner-paper]: https://www.tandfonline.com/doi/pdf/10.1207/s15326985ep4102_1
```

**Comment by brownsarahm** [comment]
Thanks @robitalec! I made a minor update to the link, but this looks good to me. 

@carpentries/trainer-training-maintainers do we have a preferred type of link and/or a way to confirm that links are not behind paywall ( I have no idea when my computer does magic to give me access vs not)

**Comment by karenword** [comment]
I don't see a paywall here. As far as I'm concerned anything that is open access is fine and things behind paywalls may have to be fine in some cases.

**Comment by robitalec** [comment]
Thank you both! Good to know for links/paywalls.

**Pull request merged**